test/integration/targets/inventory_vmware_host_inventory/ansible.cfg

```diff
@@ -0,0 +1,8 @@
+[defaults]
+inventory = test-config.vmware.yaml
```

Comment:
This seems unnecessary to me, as you are always passing with -i on runme.sh anyway.